docs/resources/widgets/pages/how-tos.md

```diff
@@ -0,0 +1,481 @@
+---
+title: How-to
```

Comment:
I would give it a more descriptive name like "How to use pages"

Reply:
This page has a lot of information in it. But I worry we separated out the concepts and how-tos too much. It feels like we need to mesh them together more. I think we need to take another stab at organizing this information in a better way. 

For example, maybe having one page thats like "getting started with pages" where we describe what a page is / the scaffold thing, etc. and then show how to create a page, and configure it. 

Next page could be "navigating across pages" where we talk a bit about navigation and then the how-tos for navigation

... basically I think we give too much conceptual information and then all the how-tos here, it makes it hard to grasp the concepts and then apply them

Reply:
PLus I think there's just too much on this page, we need to break it up a bit